extensions/reactive-datasource/runtime/src/main/java/io/quarkus/reactive/datasource/runtime/DataSourceReactiveRuntimeConfig.java

```diff
@@ -24,4 +28,58 @@
      */
     @ConfigItem
     public OptionalInt maxSize;
+
+    /**
+     * Whether all server certificates should be trusted.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean trustAll;
+
+    /**
+     * Trust configuration in the PEM format.
+     * <p>
+     * When enabled, {@link #trustCertificateJks} and {@link #trustCertificatePfx} must be disabled.
```

Comment:
This javadoc is used in the configuration reference doc. Thus it's better to avoid link to properties and also use the "dash case" versions that are used in config (so `trust-certificate-jks` for instance).

Reply:
Done